package.json

```diff
@@ -19,13 +19,13 @@
     "@commitlint/config-conventional": "17.2.0",
     "@jscutlery/semver": "2.29.3",
```

Comment:
`jscutlery/semver` was just [updated to support nx 16 via 3.0.0](https://github.com/jscutlery/semver/releases)